controller/src/test/java/io/pravega/controller/server/eventProcessor/RequestHandlersTest.java

```diff
@@ -840,6 +841,7 @@ public long getRequestId() {
         future.join();
     }
 
+    @SneakyThrows
```

Comment:
Why do you need this? You could add the exception in the test signature itself.

Reply:
used throws Exception